oaknational/Oak-Web-Application
# fix(get-sanity-client): loadEnvConfig depending on NODE_ENV

## Description

- updates use of `@next/env` to respect NODE_ENV

## How to test

1. Remove all env from `.env` other than `GOOGLE_SECRET_MANAGER_SERVICE_ACCOUNT` and `OAK_CONFIG_LOCATION`.
1. run `npm run dev`
2. App should run fine
3. Should see `Loaded env from .env.development.local` in terminal logs
